### PR TITLE
cleaned up comment router, auth router and user router

### PR DESCRIPTION
**Pull request by marieglr**
- got rid of nested promises in comments router
- eliminated fields enumeration and repetition in sign up route
- eliminated field enumaration in update settings route, and corrected a bug that made that encrypted password wasnt updated
- added my email to the contact button on unverified page :)

### CODE REVIEW

File: routes/auth-router.js [hunk 1]
```diff
@@ -36,59 +33,27 @@ router.get("/signup", (req, res, next)=>{
...
-    course,
-    courseTimeStructure,
-    IronhackCourseCity,
-    cohortTime,
-    currentCity,
-    employmentStatus,
-    currentCompany
-  })
+  fields.encryptedPassword = bcrypt.hashSync(fields.originalPassword, 10);
+  User.create({ ...fields, pictureURL: secure_url})
```
Comment (mpligioso) [inline routes/auth-router.js]: Does this work when a user doesn't add a photo? Or is that still required?

Comment (marieglr) [reply routes/auth-router.js]: It was already required before I refactored it, but i can add an if statement if you think it is better not to make it mandatory!

Comment (mpligioso) [reply routes/auth-router.js]: If I remember correctly, it throws a nice old error if you sign up without, so for user experience we should probably change that :+1:

Comment (marieglr) [reply routes/auth-router.js]: yes ma'am !

**Review by mpligioso — APPROVED** [review]
👍 perfecto

**Pull request merged**